cleanup/remove-skeleton-databases.sh

```diff
@@ -0,0 +1 @@
+remove-zombies.sh
```

Comment:
there is some usage information within this file naming "zombies" instead of "skeleton" now